client/src/components/Admin/UserList.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <table class="table">
+    <thead>
+      <tr>
+        <th scope="col">#</th>
+        <th scope="col">Firstname</th>
+        <th scope="col">Lastname</th>
+        <th scope="col">Gender</th>
+        <th scope="col">Country</th>
+        <th scope="col">DOB</th>
+        <th scope="col">Email</th>
+        <th scope="col">Designation</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr v-for="(userDetails, index) in users" :key="index">
+        <td>{{ index + 1 }}</td>
+        <td>{{ userDetails.firstName }}</td>
+        <td>{{ userDetails.lastName }}</td>
+        <td>{{ userDetails.gender }}</td>
+        <td>{{ userDetails.country }}</td>
+        <td>{{ new Date(userDetails.dob).toLocaleDateString() }}</td>
```

Comment:
its always better to use method while rendering, avoid direct overriding of data.
create a new method, pass `userDetails.dob` as parameter and then return the formatted date.